File: app/src/main/java/es/etologic/mahjongscoring2/domain/model/PlayerRanking.java
package es.etologic.mahjongscoring2.domain.model;

import android.support.annotation.NonNull;

public class PlayerRanking implements Comparable<PlayerRanking> {

    private final String name;
    private String points;
    private final String score;

    public String getName() {
        return name;
    }

    public String getPoints() {
        return points;
    }
    public void setPoints(String points) {
        this.points = points;
    }

    public String getScore() {
        return score;
    }

    public PlayerRanking(String name, String score) {
        this.name = name;
        this.score = score;
    }

    @Override public int compareTo(@NonNull PlayerRanking o) {
        return score.compareTo(o.getScore());
    }
}

File: app/src/main/java/es/etologic/mahjongscoring2/domain/model/PlayerRanking.kt
package es.etologic.mahjongscoring2.domain.model

class PlayerRanking(val name: String, val score: String) : Comparable<PlayerRanking> {
    
    var points: String? = null
    
    override fun compareTo(other: PlayerRanking): Int = score.compareTo(other.score)
}
